File: tests/machine/x/dart/go_auto.dart

class testpkg {
  static int Add(int a, int b) => a + b;
  static const double Pi = 3.14;
  static const int Answer = 42;
}

void main() {
  print(testpkg.Add(2, 3));
  print(testpkg.Pi);
  print(testpkg.Answer);
}

File: tests/machine/x/rust/go_auto.rs
fn main() {
    mod testpkg {
        pub fn Add(a: i32, b: i32) -> i32 { a + b }
        pub const Pi: f64 = 3.14;
        pub const Answer: i32 = 42;
    }
    println!("{:?}", testpkg::Add(2, 3));
    println!("{:?}", testpkg::Pi);
    println!("{:?}", testpkg::Answer);
}
